app/ResolveRoute.js

```diff
@@ -7,8 +7,9 @@ export const routeRegex = {
     CategoryFilters: /^\/(hot|created|trending|active|promoted)\/?$/ig,
     PostNoCategory: /^\/(@[\w\.\d-]+)\/([\w\d-]+)/,
     Post: /^\/([\w\d\-\/]+)\/(\@[\w\d\.-]+)\/([\w\d-]+)\/?($|\?)/,
+    PostJson: /^\/([\w\d\-\/]+)\/(\@[\w\d\.-]+)\/([\w\d-]+)(\.json)$/,
     UserJson: /^\/(@[\w\.jso\d-]+)(\.json)$/,
```

Comment:
@roadscape this is different and adds .json routes for posts, the regex is used in post_json.jsx to catch the requests.